.github/workflows/release.yml

```diff
@@ -44,23 +44,23 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
-      - name: Get latest version of stable Rust
-        run: rustup update stable
+      - name: Get latest version of nightly Rust
+        run: rustup update nightly
       - name: Install target
         run: rustup target add ${{ matrix.arch }}
       - uses: Swatinem/rust-cache@v2
         with:
           cache-on-failure: false
-      - name: Run cargo clean
-        run: cargo clean
+      - name: Run cargo +nightly clean
+        run: cargo +nightly clean
 
       # ==============================
       #       Builds
       # ==============================
 
       - name: Install cross from source
         run: |
-          cargo install --git https://github.com/cross-rs/cross cross --branch main
+          cargo +nightly install --git https://github.com/cross-rs/cross cross --branch main
```

Comment:
Please add Verification Evidence to the Pull Request description showing that releases work end-to-end with Nightly builds. We typically do this by forking and running a release in a forked branch

Reply:
https://github.com/AntonD3/era-test-node/actions/runs/6507401464